src/components/MainWrapper/index.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex"
+  },
+  content: {
+    maxWidth: "60%",
```

Comment:
Looks great! Just thinking do we want to go with 60% for all or something a little smaller? Maybe one for you @elishkaflint

Reply:
also, I wonder if we want a mobile breakpoint the is 100%?